attributes/default.rb

```diff
@@ -19,3 +19,12 @@
 #
 
 default['ms_dotnet']['timeout'] = 600
+
+if platform? 'windows'
+  nt_version = node['platform_version'].to_f
+  if nt_version >= 6.2
+    default['ms_dotnet']['feature']['enable_all']            = true
```

Comment:
Currently this option is only used for .NET3.5; if we start to use a similar system for other .NET version I think it could be usefull to have multiple attributes offering us better control.

To keep a coherent structure for attribute file I would even ask you to move your attribute to a `ms_dotnet3` attribute file.

Sorry if you find me a bit strict on this point :)